.bin/esy

```diff
@@ -27,30 +27,69 @@ SCRIPTDIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
 # stderr to nonblocking mode
 
 EJECT_PATH="$PWD/node_modules/.cache/esy"
+ENV_PATH="$PWD/node_modules/.cache/esy-env"
```

Comment:
This is going to be great for editor/IDE integration. In fact, this can become a language/build-system agnostic convention.

cc @freebroccolo.